corsikaio/subblocks/event_end.py

```diff
@@ -39,10 +39,17 @@
 event_end_dtype_65 = build_dtype(event_end_fields_65)
 event_end_dtype_7x = build_dtype(event_end_fields_7x)
 
+event_end_thin_dtype_65 = build_dtype(event_end_fields_65, itemsize = 4 * 312)
```

Comment:
Use the constant you defined for this

Reply:
Do you mean to import BLOCK_SIZE_BYTES_THIN from ..constants and use that to define the itemsize? In that case should I also implement the same (BLOCK_SIZE_BYTES) for the itemsize in build_dtype method of dtypes.py?